packages/grid_client/src/zos/znet.ts

```diff
@@ -55,4 +55,4 @@ class Znet extends WorkloadData {
   }
 }
 
-export { Znet, Peer };
+export { Znet, Peer, Mycelium };
```

Comment:
why do you export mycelium here? Is this change necessary?

Reply:
It's necessary as we use it here https://github.com/threefoldtech/tfgrid-sdk-ts/blob/7974177e30c01e02f9b6429fdf418515cd63bde9/packages/grid_client/tests/modules/workload.test.ts#L48